change/@microsoft-fast-foundation-fb2bff9c-3129-4433-bf60-2b85c82258d4.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix(design-system): don't allow FoundationElement to be defined directly",
```

Comment:
I think I'm reading this wrong, but this reads as we are *preventing* - should we drop "don't" in favor of just "allow FoundationElement..." so it reads as this is supported*.

Reply:
I mean, you can no longer have this happen: `customElements.define('foo-bar', FoundationElement)`. Not sure the best way to worth that.